omod/src/main/web/dashboardwidgets/visitbyencountertype/visitbyencountertype.html

```diff
@@ -1,11 +1,11 @@
-<ul>
-    <li ng-if="$ctrl.visits.length != 0" ng-repeat="visit in $ctrl.visits track by $index">
-        <a href="{{$ctrl.serverUrl}}/coreapps/patientdashboard/patientDashboard.page?patientId={{$ctrl.config.patientUuid}}&visitId={{visit.uuid}}">
-            {{visit.startDatetime | date: $ctrl.config.dateFormat}}
-        </a>
-        <div class="tag" ng-if="visit.encounterType !== ''">{{visit.encounterType}}</div>
-    </li>
-    <p ng-if="$ctrl.visits.length == 0">
-        None
-    </p>
+<ul>
+    <li ng-if="$ctrl.visits.length != 0" ng-repeat="visit in $ctrl.visits track by $index">
+        <a href="{{$ctrl.serverUrl}}/coreapps/patientdashboard/patientDashboard.page?patientId={{$ctrl.config.patientUuid}}&visitId={{visit.uuid}}">
+            {{visit.startDatetime | date: $ctrl.config.dateFormat}}
+        </a>
+        <div class="tag" ng-if="visit.encounterType !== ''">{{visit.encounterType}}</div>
+    </li>
+    <p ng-if="$ctrl.visits.length == 0">
+    	{{$ctrl.noDataMessage}}

```

Comment:
it's still unclear everywhere else what's changing in this commit except here yet even here there's mis-formatting. that makes it heard for your code to entice reviews @moshonk

Reply:
Hi @kaweesi, seems like my instance of eclipse is messing around with the code formatting. I'm looking into it.